staff/alejandro-perez/cv/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<style> html{
+    background-color: azure;
+} </style>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV</title>
+</head>
+<body>
+    <h1>Pepe el del precurso</h1>
+    <h2>About me</h2>
+    <ul>
+        <li style="list-style: none">    <img src="./descarga.jpeg" alt="neng"></li>
```

Comment:
Give the title a more descriptive name, like your own -- "CV" is a little generic